### PR TITLE
Updates with final CVEs

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Final CVEs in
CVE added to page title for clarity
New changes marked with updated date updated
Removed wrong format date for Updated note
Reordered all the indexes to descending CVE

### CODE REVIEW

File: advocacy_docs/security/advisories/cve202341114.mdx [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-title: EDB Postgres Advanced Server (EPAS) authenticated users may fetch any URL
-navTitle: CVE-2023-XXXXX-6
+title: CVE-2023-41114 - EDB Postgres Advanced Server (EPAS) authenticated users may fetch any URL
+navTitle: CVE-2023-41114
 ---
 
 First Published: 2023/08/21
 
-Last Updated: 2023/08/21
+Last Updated: 2023/08/28
```
Comment (drothery-edb) [inline advocacy_docs/security/advisories/cve202341114.mdx:8]: do we want to mention what the update was about?

File: advocacy_docs/security/advisories/cve202341116.mdx [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
-title: EDB Postgres Advanced Server (EPAS) permission bypass for materialized views
-navTitle: CVE-2023-XXXXX-5
+title: CVE-2023-41116 - EDB Postgres Advanced Server (EPAS) permission bypass for materialized views
+navTitle: CVE-2023-41116
 ---
 
 First Published: 2023/08/21
 
-Last Updated: 2023/08/21
+Last Updated: 2023/08/28
```
Comment (drothery-edb) [inline advocacy_docs/security/advisories/cve202341116.mdx:8]: same comment

File: advocacy_docs/security/advisories/cve202341119.mdx [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-title: EDB Postgres Advanced Server (EPAS) dbms_aq helper function may run arbitrary SQL as a superuser
-navTitle: CVE-2023-XXXXX-2
+title: CVE-2023-41119 - EDB Postgres Advanced Server (EPAS) dbms_aq helper function may run arbitrary SQL as a superuser
+navTitle: CVE-2023-41119
 ---
 
 First Published: 2023/08/21
 
-Last Updated: 2023/08/21
+Last Updated: 2023/08/28
```
Comment (drothery-edb) [inline advocacy_docs/security/advisories/cve202341119.mdx:8]: reason?

File: advocacy_docs/security/advisories/cve202341114.mdx (hunk 1 above)
Comment (djw-m) [reply advocacy_docs/security/advisories/cve202341114.mdx:8]: In the change history at the bottom.

File: advocacy_docs/security/advisories/cve202341118.mdx [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
-title: EDB Postgres Advanced Server (EPAS) UTL_FILE permission bypass
-navTitle: CVE-2023-XXXXX-4
+title: CVE-2023-41118 - EDB Postgres Advanced Server (EPAS) UTL_FILE permission bypass
+navTitle: CVE-2023-41118
 ---
 
 First Published: 2023/08/21
 
-Last Updated: 2023/08/21
+Last Updated: 2023/08/28
```
[the remark this answers is not in the document]
Comment (djw-m) [reply advocacy_docs/security/advisories/cve202341118.mdx:8]: In the change history at the bottom.

File: advocacy_docs/security/advisories/cve202341119.mdx (hunk 3 above)
Comment (djw-m) [reply advocacy_docs/security/advisories/cve202341119.mdx:8]: In the change history at the bottom.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**